src/style.css

```diff
@@ -1,3 +1,58 @@
+:root {
+  font-family: "Roboto", sans-serif;
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+
+}
+
+.logo__link {
+  margin-left: 50px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation {
+  justify-content: center;
+  margin-right: 40px;
```

Comment:
as I mentioned above, let's remove this margin, and set `padding: 0 50px` for header